Initial folder structure

ElvaRivle opened the pull request.

ElvaRivle committed:
npm init and express added

ElvaRivle committed:
entry point js file

ElvaRivle committed:
important folders added

ElvaRivle committed:
express hello world example

becarselman commented:
Everything looks great, but I would like to have nodemon installed on this setup.

TarikVelic reviewed (approved):
Everything looks good as far as I'm concerned.

ElvaRivle committed:
nodemon and script `dev` added

installed nodemon locally as development dependency

so we must use npm run to use it

ElvaRivle commented:
> but I would like to have nodemon installed on this setup.

Done. Added nodemon as development dependency. Because it is installed locally, we need to run it via `npm run`. `dev` script in package.json added
